src/main/java/org/openrewrite/java/spring/boot3/AddMappingPathsWithTrailingSlash.java

```diff
@@ -0,0 +1,145 @@
+package org.openrewrite.java.spring.boot3;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.SourceFile;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.PartProvider;
+import org.openrewrite.java.tree.*;
+
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static org.openrewrite.java.tree.Space.EMPTY;
+
+public class AddMappingPathsWithTrailingSlash extends Recipe {
+    private static final String GET_ANNOTATION_TYPE = "org.springframework.web.bind.annotation.GetMapping";
+    private static final String REQUEST_ANNOTATION_TYPE = "org.springframework.web.bind.annotation.RequestMapping";
+    private static final String POST_ANNOTATION_TYPE = "org.springframework.web.bind.annotation.PostMapping";
+    private static final String PUT_ANNOTATION_TYPE = "org.springframework.web.bind.annotation.PutMapping";
+    private static final String PATCH_ANNOTATION_TYPE = "org.springframework.web.bind.annotation.PatchMapping";
+    private static final String DELETE_ANNOTATION_TYPE = "org.springframework.web.bind.annotation.DeleteMapping";
+    @Nullable private static J.NewArray twoStringsArrayTemplate;
+    @Nullable private static J.Assignment valueAssignmentTemplate;
+
+    @Override
+    public String getDisplayName() {
+        return "Declare the additional route explicitly on the controller handler";
```

Comment:
This display name does not do a great job of describing what this recipe does. Maybe something like this?

```suggestion
        return "Add trailing slash to Spring routes";
```